Specifications/Language/3_Expressions/ItemAccessExpressions.md

```diff
@@ -38,17 +40,19 @@ The contained items can be accessed via their name or by deconstruction, illustr
 ```
 
 The item access operator (`::`) retrieves named items.
-While named items can be accessed by their name or via deconstruction, anonymous items can only be accessed by the latter. Since deconstruction relies on all of the contained items, the usage anonymous items is discourage when these items need to be accessed outside the compilation unit in which the type is defined. 
+While named items can be accessed by their name or via deconstruction, anonymous items can only be accessed by the latter. Since deconstruction relies on all of the contained items, the usage anonymous items is discouraged when these items need to be accessed outside the compilation unit in which the type is defined. 
```

Comment:
```suggestion
While named items can be accessed by their name or via deconstruction, anonymous items can only be accessed by the latter. Since deconstruction relies on all of the contained items, the usage of anonymous items is discouraged when these items need to be accessed outside the compilation unit in which the type is defined. 
```